greenswitch/esl.py

```diff
@@ -426,8 +429,15 @@ def listen(self):
         self.server.listen(100)
 
         while True:
+            if self.connection_pool.full():
+                logging.info('Have no available slots to allocate this client connection pool reached %s' %
```

Comment:
Message can be `Unable to connect. Reached max. number of connections (max: %s).`.

Reply:
Thanks

Reply:
Correct, @italorossi. So perhaps this can be shorter...
 'Rejecting call, server is at full capacity. (max: %s).'

Reply:
s/have/there are/

Reply:
Excuse me and ignore my last comment